### PR TITLE
Added PS4 7.50 7.51 7.55 Offsets

### PR DESCRIPTION
**Pull request by EchoStretch**
Only 7.55 has been confirmed. From what I looked at 7.50 shares same offsets.

### CODE REVIEW

File: offsets.py [hunk 1]
```diff
@@ -3,6 +3,99 @@
 # This software may be modified and distributed under the terms
 # of the MIT license.  See the LICENSE file for details.
 
+# FW 7.50 / 7.51 / 7.50
+class OffsetsFirmware_750_755:
+    PPPOE_SOFTC_LIST = 0xFFFFFFFF8433FCD0
```
Comment (TheOfficialFloW) [inline offsets.py:8]: Use lower case hex decimals

File: stage1/offsets.h [hunk 2]
```diff
@@ -8,8 +8,35 @@
...
+#define kdlsym_addr_setidt 0xFFFFFFFF825D9440
+
+#define kdlsym_addr_kernel_map 0xFFFFFFFF843405B8
+#define kdlsym_addr_kmem_alloc 0xFFFFFFFF823753E0
+
+#define kdlsym_addr_kproc_create 0xFFFFFFFF8220D8F0
+#define kdlsym_addr_kproc_exit 0xFFFFFFFF8220DB60
+
+#define kdlsym_addr_ksock_create 0xFFFFFFFF82521DA0
+#define kdlsym_addr_ksock_close 0xFFFFFFFF82521E10
```
Comment (TheOfficialFloW) [inline stage1/offsets.h:32]: Lower case here and elsewhere

Comment (EchoStretch) [reply stage1/offsets.h]: Fixed

File: offsets.py (hunk 1 above)
Comment (EchoStretch) [reply offsets.py]: Fixed

**Comment by TheOfficialFloW** [comment]
Tip for the future: don't use the same commit msg 20x

**Pull request merged**